silabs-multiprotocol/config.yaml

```
@@ -1,7 +1,7 @@
 ---
-version: 2.4.5
+version: 2.4.6
 slug: silabs_multiprotocol
-name: Silicon Labs Multiprotocol
+name: "[Deprecated] Silicon Labs Multiprotocol"
```

Turns out that this actually reorders the add-ons in the store, making the deprecated ones appear first :see_no_evil: 

Let's move the deprecation information to the end:

```suggestion
name: "Silicon Labs Multiprotocol [deprecated]"
```